tst/testinstall/alghom.tst

```diff
@@ -1,12 +1,29 @@
-#@local A,B,C,ExampleRing,I,O,P,Q,R,T,b,coker,f,gensq,id,ker,m1,m2,map,pols
-#@local pr,q,x,y,z
 #############################################################################
 ##
-#W  alghom.tst                  GAP library                     Thomas Breuer
+##  This file is part of GAP, a system for computational discrete algebra
+##
+##  SPDX-License-Identifier: GPL-2.0-or-later
+##
+##  Copyright of GAP belongs to its developers, whose names are too numerous
+##  to list here. Please refer to the COPYRIGHT file for details.
 ##
+##  GAP is free software; you can redistribute it and/or modify it under
+##  the terms of the GNU General Public License as published by the Free
+##  Software Foundation; either version 2 of the License, or (at your
+##  option) any later version.
 ##
-#Y  Copyright (C)  1998,  Lehrstuhl D für Mathematik,  RWTH Aachen,  Germany
+##  GAP is distributed in the hope that it will be useful, but WITHOUT
+##  ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+##  or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public
+##  License for more details.
 ##
+##  You should have received a copy of the GNU General Public License
+##  along with GAP; if not, write to the Free Software Foundation, Inc.,
+##  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
```

Comment:
We probably don't need copyright / license headers in `.tst` files, but they also won't hurt... I'd have a small preference for just removing them everywhere.